util/download_data.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const _ = require('lodash');
+const async = require('async');
+const child_process = require('child_process');
+const fs = require('fs-extra');
+const config = require( 'pelias-config' ).generate(require('../schema'));
+const logger = require('pelias-logger').get('download');
+
+if (require.main === module) {
+  download((err) => {
+    if (err) {
+      logger.error('Failed to download data', err);
+      process.exit(1);
+    }
+    logger.info('All done!');
+  });
+}
+
+function download(callback) {
+  let sources;
+
+  if (!_.isEmpty(config.imports.openstreetmap.download)) {
```

Comment:
if you swap the two clauses of this if/else you can get rid of the negation operator and the code reads slightly cleaner. plus we need to save those exclamation points for when we are excited about something! :P